### ISSUE TITLE
AssertionError on IfExp

### ISSUE THREAD

**Issue by c8y3** [mschwager/cohesion#20]
With version 1.1.0, cohesion throws an exception on this code:
```
class C():

    def a(self):
        print('a')
    
    def b(self):
        print('b')
    
    def f(self, choice):
        (self.a if choice else self.b)()

c = C()
c.f(True)
c.f(False)
```
Thank you.

**Comment by mschwager** [comment]
Interesting, I will look into it. Thanks for the bug report!

**Comment by mschwager** [comment]
I fixed this and released version `1.2.0` with the fix: https://pypi.org/project/cohesion/1.2.0/
